.ci-support/production-setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 set -x
-PRODUCTION_CHANGE_OWNER="illinois-ceesd"
+PRODUCTION_CHANGE_OWNER=""
```

Comment:
Because this branch merges clean with y1-production, we don't need to change y1-production to make it compatible. It really doesn't matter what this is set to, because it doesn't even look at this unless the PRODUCTION_CHANGE_BRANCH is set.